buildSrc/subprojects/build/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 dependencies {
-    api("com.google.guava:guava:26.0-jre")
+    api("com.google.guava:guava:27.1-jre")
```

Comment:
Why are we using the `jre` variant of Guava in `buildSrc` and the `android` variant in the main build?

Reply:
We should be using the `jre` variant everywhere, but we can't currently use it in the production code, because of the change in how `ImmutableSet.Builder` works: https://github.com/google/guava/issues/3223.